Shop.BLL/Common/Exceptions/BadRequestExceptions/BadRequestException.cs

```diff
@@ -0,0 +1,4 @@
+namespace Shop.BLL.Common.Exceptions.BadRequestExceptions
```

Comment:
it is better to remove the common folder, it is not needed and write all exceptions immediately in the Exception folder, without creating additional ones